pandas/compat/pickle_compat.py

```diff
@@ -54,6 +55,22 @@ def load_reduce(self):
         raise
 
 
+class _LoadSparseSeries:
+    # To load a SparseSeries as a Series[Sparse]
+    def __new__(cls) -> Any:
```

Comment:
Also w.r.t annotation could probably just use `FrameOrSeries` from pandas._typing, as that is tied to subclasses of NDFrame (applicable to both methods)